x-pack/plugins/osquery/server/routes/pack/create_pack_route.ts

```diff
@@ -150,7 +152,7 @@ export const createPackRoute = (router: IRouter, osqueryContext: OsqueryAppConte
                     }
 
                     set(draft, `inputs[0].config.osquery.value.packs.${packSO.attributes.name}`, {
-                      shard: policyShards[packagePolicy.policy_ids[0]]
+                      shard: policyShards[packagePolicy.policy_ids[0]] // TODO
```

Comment:
@ashokaditya I'm not sure what would be the logic here if there are multiple `policy_ids`